src/group_theory/index.lean

```diff
@@ -70,6 +70,10 @@ end
 eq.trans (congr_arg index (by refl))
   ((H.subgroup_of f.range).index_comap_of_surjective f.range_restrict_surjective)
 
+@[to_additive] lemma index_map {G' : Type*} [group G'] {f : G →* G'}
```

Comment:
I think `hf1` implies that the RHS divides the LHS, while `hf2` implies the converse. Could you please prove those lemmas? `dvd_index_map`(`_of_surjective`) and `index_map_dvd`(`_of_ker_le`), and then prove this lemma using `dvd_antisymm`.

Reply:
Nice!